pythonforandroid/bootstraps/common/build/build.py

```diff
@@ -529,8 +529,9 @@ def make_package(args):
         aars=aars,
         jars=jars,
         android_api=android_api,
-        build_tools_version=build_tools_version
-        )
+        build_tools_version=build_tools_version,
+        is_library=get_bootstrap_name() == 'service_library',
```

Comment:
Could you make this `is_library=(get_bootstrap_name() == 'service_library'),`, I think it's clearer.

Reply:
done